src/encode.rs

```diff
@@ -61,6 +63,11 @@ impl From<u64> for OrePlaintext<u64> {
     }
 }
 
+impl From<String> for OrePlaintext<u64> {
```

Comment:
Isn't `Plaintext` defined in terms of `N` where `N` is the number of bytes? Or is this different to the `Plaintext` used in ore.rs?

Reply:
One other consideration here is that there will be multiple ways to do this conversion depending on the goal. For orderable strings we might use a compressed or even lossy conversion. Could these traits be overloaded to do that? Maybe we don't need to worry for now but just flagging it.

Reply:
Yeah, this is the [other plaintext type](https://www.notion.so/cipherstash/ore_encoding-rs-should-use-ore-rs-PlainText-type-b00d3efa25014c2394a1ad71dc1efad0#9160fbd5f1d949e4801d1bb123b456c1) in action.  Presumably ordered strings or whatever else could go into a different destination type, or something.

This code is a straight-up rip from `node-ore-rs`, pulled in here so I don't have to play copy-pasta games in ruby-ore-rs, BTW.

Reply:
Understood.